docker-compose.codex.yml

```diff
@@ -225,10 +225,11 @@ services:
       retries: 60
     volumes:
       - .dockerfiles/dev-frontend/docker-entrypoint.sh:/docker-entrypoint.sh
-      - ./_frontend.codex:/code
+      - ./:/code
```

Comment:
So I am used to developing locally with a diff here, something like 
```
/path/to/my/repo:/code
```
Do I need to move my repo into a `code/_frontend` subfolder now? Or what would be the best way to do it

Reply:
Two ways to solve this.
1. change the `working_dir` setting in this service.
2. adjust the entrypoint script to be somewhat aware of the situation.

I've gone with option two because it means one less change the developer would need to fiddle with. So I've adjusted the script to check for the existence of the submodule directory. If the directory exists, it'll change to to it, otherwise it'll assume the code is in the current working directory.

Reply:
Oh that's great!

Reply:
I'll approve but I'd wait for another approval from a backend person